java/ql/src/Security/CWE/CWE-829/InsecureDependencyResolution.ql

```diff
@@ -0,0 +1,38 @@
+/**
+ * @name Failure to use HTTPS or SFTP URL in Maven artifact upload/download
+ * @description Non-HTTPS connections can be intercepted by third parties.
+ * @kind path-problem
+ * @problem.severity error
+ * @precision very high
```

Comment:
```suggestion
 * @precision very-high
```
This is required to ensure a valid [`@precision`](https://help.semmle.com/QL/learn-ql/writing-queries/query-metadata.html) value that can be interpreted correctly by LGTM.

Reply:
Just spotted an error in our documentation related to this. See #2452.